dev-python/ordered-set/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+		<name>Python</name>
+	</maintainer>
```

Comment:
You can't add the python project here without their consent.

Reply:
Since these new packages are dependencies of line_profiler, I was hoping that whoever was maintaining line_profiler would be able to maintain these new dependencies.

However I did just notice that it is not the python project that is maintaining the line_profiler ebuild, it is the science project. @heroxbd you are on the science team, right? Would it be okay to add the science project to the metadata files here? (Scikit-build does sound like a science package anyway :P )

Reply:
> However I did just notice that it is not the python project that is maintaining the line_profiler ebuild, it is the science project. @heroxbd you are on the science team, right? Would it be okay to add the science project to the metadata files here? (Scikit-build does sound like a science package anyway :P )

No problem, you can add the science project to line_profiler and its dependencies.